Docs: Added bookmarklet method to import CKEditor 5 inspector (#12118)
ckeditor/ckeditor5: docs/framework/guides/development-tools.md

<script src="../node_modules/@ckeditor/ckeditor5-inspector/build/inspector.js"></script>
```

#### Inspector as a bookmarklet

> Alternatively, you might create a bookmarklet in your browser that will allow you to open inspector on every page without interference

Reviewer: ```suggestion
Alternatively, you can create a bookmarklet in your browser that will allow you to open inspector on any page without interference with its source code. 
```

> Alternatively, you might create a bookmarklet in your browser that will allow you to open inspector on every page without interference

Reviewer 2: Some "markdown-codestyle" things:

* Don't wrap the lines manually. Configure your IDE to wrap them for you.
* Replace `<b>` with `**`.
* Add extra padding (empty new lines) between blocks of content. In this case – before and after the code block.

Reviewer: Done.

>  with its source code. 

Reviewer: this line should be removed

> <b>Important note: this method won't work if the page has Content Security Policy enabled.</b>

Reviewer: ```suggestion
<b>Important note: this method will not work if the page has Content Security Policy enabled.</b>
```

To create a bookmarklet, paste this code as the URL of the newly created bookmark in the browser of your choice:
```js
javascript:(function(){let script=document.createElement('script');script.src='https://unpkg.com/@ckeditor/ckeditor5-inspector/build/inspector.js';script.onload=()=>CKEditorInspector.attachToAll();document.head.appendChild(script);})()
```

> After that, you can open CKEditor 5 inspector by pressing the previously created bookmark. 

Reviewer 2: open -> load?

Opening sounds like it's closed so you open it. While with inspector it's more to make it appear at all.

Reviewer: Also changed pressing -> using.

<script src="../node_modules/@ckeditor/ckeditor5-inspector/build/inspector.js"></script>
```

> #### Inspector as a bookmarklet

Reviewer 2: I know it sounds a bit like a sub-section to "Importing the inspector" but heading level 4 is really deeeep in the structure. So maybe l3 looks better?
